### DISCUSSION TITLE
Fixing the broken link in the second search provided by htmx request

### DISCUSSION THREAD

**Discussion by brandondelpozo** [codingforentrepreneurs/Try-Django-3.2#10] [Ideas]
Hi everyone I have an improvement for  Try-Django-3.2/templates/search/partials/results.html , if you change **`< a href='/search/?q={{ request.GET.q }}&type={{ request.GET.type }}'>View All</a>`**  for just "**`< a href='{{ request.get_full_path }}'>View All</a>`** that will help you fix the broken link in the second search provided by htmx request. I hope that helps you improve your search flow, have a great day!

Brandon

**Comment by jmitchel3** [comment]
Can you use the _markdown_ syntax for the links `https://github.com/search/?q={{ request.GET.q }}&type={{ request.GET.type }}` ?

**Reply by brandondelpozo** [reply]
I'm sorry this is my first time using discussions on github, I just did it :)
